### DISCUSSION TITLE
[Scam Alert] Trading Broker Coinlocally

### DISCUSSION THREAD

**Discussion by Blockunity** [Blockunity/WarningList#10] [General]
نام : Coinlocally
سایت: www.coinlocally.com

توضیحات:
وب سایت کوین لوکالی که با استفاده از API بایننس و طراحی کاملا شبیه سایت بایننس تلاش دارد در بین کامیونیتی فارسی زبان جای باز کند.
مبلغان اصلی این صرافی اقای هلاکویی بوده برای دوره تورنومت ترید:
https://t.me/didebanmarket/120
![Screenshot_20220720-001531_Telegram.jpg](https://user-images.githubusercontent.com/72666334/179857680-ff8f0363-7f45-4029-92c9-5a950c5668e3.jpg)
این صرافی طی بررسی های انجام شده بیشتر کاربر هاش را از IP ایران دارد:
 https://www.similarweb.com/website/coinlocally.com/#geography
در دیتای داخل سایت میتونید ببینید که منبع بزرگ ارجاع دهنده اینستاگرام هست و همچنین بیشتر کاربر ها سابقه جستوجو دیجی کالا را هم دارند.

در ادامه بررسی به فردی رسیدیم که بیزینس دولوپمنت این بروکر هست به اسم مهرداد لاجوردی:
https://www.linkedin.com/in/mehrdad-lajevardi-86b6b437/?originalSubdomain=ir
**در سابقه کاری ایشون دو شرکت پارس روبین و تامین ارتباطات تبیان توجه مارا جلب کرد.**
شرکت پارس روبین در گذشته سابقه فعالیت در حوزه مالی و ترید داشته.
شرکت تبیان نیز زیر نظر سازمان سپاه و قوه قضاییه فعالیت داره!

نتیجه گیری:

**با توجه به بررسی های انجام از ترافیک سایت و نامعلوم بودن تیم و سازنده این بروکر و همچنین نامعلوم بودن مکان این بروکر و همچنین نداشتن احراز هویت که از طرف بایننس برای تمامی بروکر ها اجباری هست !**
با در کنار هم گذاشتن اطلاعات استخراج شده به این نتیجه رسیدیم که این بروکر ایرانی هست و هیچ ارتباطی با بایننس بجز داشتن API ندارد .
امنیت این بروکر بسیار پایین هست و امکان اسکم شدن با درصد بسیار بالایی دارد.
**مبلغان این بروکر جز تبلیغ و دادن کد معرف و به خطر انداختن سرمایه شما قصد کمک ندارند و از این افراد حتما دوری به عمل اورید.**

آپدیت می‌شود

**Comment by Blockunity** [comment]
[https://coinlocally.com/policy](url)

طبق ادعای سایت به قوانین FATF پایبنده در صورتی که با کشور های تحت تحریم مثل ایران در حال خدماتدهی هست. این خودش نقش قوانین خود سایت هست.

<img width="883" alt="Screenshot 2022-07-26 000439" src="https://user-images.githubusercontent.com/72666334/180882370-e46e9516-d096-4a37-8529-63eb89c12575.png">

**Comment by DiabloNova** [comment]
مهرداد لاجوردی ( به نام مستعار مهرداد کاشی) در سال ۲۰۰۵ به همراه یکی از اعضای سپاه پاسداران و  احمد علیرضا بیگی و علی جعفری ( رییس دفتر وزیر نفت در دولت اول احمدی نژاد) در کشور امارات شرکتی را به نام Lamorada General Trading LLC راه اندازی میکنند .  در آن زمان مهرداد لاجوردی یکی از افراد فعال سپاه در بخش تجارت خارجی بود. این شرکت در زمینه تجارت بین الملل فعالیت میکند . در سال ۲۰۱۸ این شرکت با راه اندازی وب سایت [coinlocally.info](http://coinlocally.info/) بر روی یک سرور داخل ایران اقدام به ورود به حوزه ارزهای رمزگذاری شده میکند. اما به دلیل نبود دانش لازم در این مجموعه، قراردادی را با یک شرکت هندی به نام white label exchange software  برای توسعه این وبسیات منعقد میکنند.
اگر به سورس کد وبسایت حال حاضر کوین لوکالی مراجعه کنید متوجه میشوید که این وبسایت از آی فریم استفاده میکند ،در واقع وبسایت کوین لوکالی همین الان هم بر روی [coinlocally.info](http://coinlocally.info/) قرار دارد که از آنجا به coinlocallydemo.comz دایرکت شده  و مجددا به سایت اصلی دایرکت شده و توسعه دهندگان هندی تصمیم گرفتند برای کاهش زحمت و هزینه‌های خود آن را به Coinlocally.com ارجاع دهند و نه انتقال. مخصوصا اینکه این وبسایت از زبان rust  برای توسعه نرم افزاری استفاده کرده که بیش از اندازه خطرناک است .

**Comment by DiabloNova** [comment]
اگر به صفحه "درباره ما"  سایت Coinlocally.info که در سایت [archive.org](http://archive.org/)  ذخیره شده مراجعه کنید ،شماره تماس و نشانی دفتر تهران! آنها را نیز درج کرده است. 
لینک صفحه [درباره ما ](https://web.archive.org/web/20220127193850/https://coinlocally.info/%D8%AF%D8%B1%D8%A8%D8%A7%D8%B1%D9%87-%D9%85%D8%A7/
)
(https://web.archive.org/web/20220127193850/https:/coinlocally.info/%D8%AF%D8%B1%D8%A8%D8%A7%D8%B1%D9%87-%D9%85%D8%A7/)

صفحه توضیحات توسعه دهنده این سایت که از نام Coinlocally by lamorada general trading llc استفاده میکند :
https://www.crashedcases.com/how-to-fix-coinlocally-by-lamorada-general-trading-llc-app-not-working-not-loading-and-other-issues/

صفحه آگهی استخدام برنامه نویس سایت کوین لوکالی در تهران به همراه شماره تماس و آدرس اینترنتی 
https://computerjobs.ir/component/jsjobs/company-detail/company-3007/nav-31?Itemid=4828

اطلاعات مربوط به معاملات اخیر شرکت lamorada general trading llc  در حوزه دارویی ، فلزی، صنایع دفاعی، صنایع شیمیایی و عناصر سنگین :
https://en.52wmb.com/buyer/140227409
در سال ۲۰۲۲ درست قبل از راه اندازی توکن کوین لوکالی، این شرکت اقدام به ساخت آزمایشی یک توکن به نام Frz Solar System و به نشانی وبسیات https://frzss.com/ میکند  صفحه این توکن در Coinmarketcap  : https://coinmarketcap.com/community/profile/Frz_Solar_System/
است ، با مراجعه به این سایت و سایت های وابسته به آن متوجه شباهت بسیار آشکار این سایت با کوین لوکالی خواهید شد . 
همچنین با مراجعه به صفحه Coinlocally  در این وبسایت ( منظور صفحه این وبسایت است و نه توکن آن):
https://coinmarketcap.com/community/profile/Coinlocally 
به لیست دنبال کننده های این صفحه که مراجعه کنید ،  مشاهده خواهید کرد که این سایت فقط ۴ کاربر را دنبال می‌کند: 
Coinlocally!, Coinmarketcap, BNB Chain
و یک حساب کاربری به نام 34345 و نام کاربری Push319 می باشند که توسعه دهنده دو توکن کوین لوکالی با نمادهای CLYC  و cLYC و FRZZ است که دو توکن کوین لوکالی به دلیل نقض قوانین کلاهبرداری این سایت مسدود شده و حساب کاربری توسعه دهنده آن هم غیر فعال شده است .
موضوع از آنجایی حساس میشود که سایت coinmarketcap  دلیل بستن نماد این سایت را فهرست ضوابط لیست B-3 اعلام کرده است :
https://coinmarketcap.com/currencies/coinlocally-coin/
لیست B-3 
در بند آخر لیست B-3  درباره نماد های حذف شده توضیح می‌دهد:
- نقدینگی کم یا فعالیت تجاری مشکوک.
- توقف توسعه و/یا فعالیت تجاری پروژه.
-لیست پروژه در این وب‌سایت نتیجه اطلاعات گمراه کننده، ناقص یا نادرست بود.
- پروژه (و/یا شرکای آن) تحت بررسی هستند، در فهرست‌های نظارتی تنظیم‌کننده، یا به دلیل نقض قانون(ها)، اساسنامه(ها)، و مقررات(ها) مجرم شناخته شده است.
- اجرای یا استقبال فوق العاده ضعیف توسط جامعه پروژه.
- نقض IP
-هر عامل دیگری که CoinMarketCap برای کاربران خود خطرناک بداند

متاسفانه وبسیات کوین لوکالی به دلیل استفاده از زبان rust  و ارجاع آن از یک سرور دیگر به این دامنه دارای مشکلات شدید امنیتی ست که نمیتوان آن را نادیده گرفت که موجب میشود دارایی کاربران در معرض خطر مسدود شدن ، به سرقت رفتن و مشکلات بالقوه‌ای که در حوصله این مبحث نیست قرار می‌دهد. 
( پی نوشت : دو سفر اخیر محمود احمدی نژاد به امارات متحده عربی نیز احتمال دارد به این موضوع ارتباط داشته باشد که نیاز به بررسی بیشتر است که فعلا مجالی برای انجام آن نیست)

**Comment by DiabloNova** [comment]
علاوه بر این ، از وبسایت باینانس درباره سایت کوین لوکالی سوال پرسیدم ، باینانس پس از بررسی اعلام کرد که این سایت حتی به عنوان بروکر هم با باینانس همکاری ندارد.
در حال حاضر بیش از دو ماه است که سایت کوین لوکالی به بهانه آپدیت زیرساخت های سایت!!!! امکانات خودش رو غیر فعال کرده. 
اگر از پشتیبانی سایت کوین لوکالی بپرسید دفتر انها کجاست، میگویند در چندین نفطه از دنیا شعبه دارند که دروغ محض است
سایت کوین لوکالی با نام Coinlocally LLC در سال 2020 در شهر ایروان ارمنستان ثبت شده، توسط پسر شهردار قبلی ایروان که به اتهام فساد مالی در تجارت با سپاه پاسداران برکنار شد. آدرسی که در ثبت این شرکت درج شده هم اکنون برای فروش یا اجاره بلند مدت تبلیغ شده است.
https://website.informer.com/NA+coinlocally+LLC.html
https://who.is/whois/coinlocally.com
